### ISSUE TITLE
Console.Writeline/Write should not count as a token.

### ISSUE THREAD

**Issue by Moonwalker316** [SebLague/Chess-Challenge#241]
The title explains it, but right now debugging causes token use. Not just that aswell. Each CHARACTER counts as a token. This is ridiculous btw. I know you can edit it out but when making your bot it's impossible to get a real number for your token usage.

**Comment by mcthouacbb** [comment]
The reason each character counts as a token is to prevent people from storing data in strings without using any tokens.

**Comment by Moonwalker316** [comment]
True, but there atleast should be a toggle Console tokens button. That way you could see your real token count. Of course the one including Console tokens will be used to judge but it would be nice to have that.

**Comment by natebursch** [comment]
https://github.com/SebLague/Chess-Challenge/issues/163
